src/providers/iris/iris.ts

```diff
@@ -97,6 +98,7 @@ export class IrisProvider {
       this.oncallTeams = [];
       this.oncallServices = [];
       this.oncallPinnedTeams = [];
+      this.loadOncallUserCache();
```

Comment:
Nontrivial initialization code lives in `ngOnInit` as a best practice. Just setting default values isn't a huge deal, but calling `loadOncallUserCache` should probably use `ngOnInit`